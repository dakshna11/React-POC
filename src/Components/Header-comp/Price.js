import React from 'react';

function Price(props) {
    return (
        <div>
            <div className="about-us">
                <div className='about-container' >
                    <h1 className="about-text">Washing Plan</h1>
                    <p>Home / Price</p>
                </div>  
            </div>
            <div>
                <p id='washing-heading'>Washing Plan</p>
                <h1 style={{textAlign: 'center', color:'#202c45', fontSize:'50px'}}>Choose Your Plan</h1>
            </div>
            <div className='price-main'>
                <div className='price1'>
                    <p id='price-heading'>BASIC CLEANING</p>
                    <p id='price-rate1'> $ 25.99</p>
                    <p>Seats Washing</p>
                    <p>Vacuum Cleaning</p>
                    <p>Exterior Cleaning</p>
                    <p>Interior Wet Cleaning</p>
                    <p>Window Wiping</p>
                    <button id='btn-price1'>Book Now</button>
                </div>
                <div className='price2'>
                    <p style={{color:'red'}} id='price-heading'>PREMIUM CLEANING</p>
                   <div id='price-rate2'><p > $</p><p> 35</p><p>.99</p></div>
                    <p>Seats Washing</p>
                    <p>Vacuum Cleaning</p>
                    <p>Exterior Cleaning</p>
                    <p>Interior Wet Cleaning</p>
                    <p>Window Wiping</p>
                    <button id='btn-about'>Book Now</button>
                </div>
                <div className='price3'>
                    <p id='price-heading'>COMPLEX CLEANING</p>
                    <p id='price-rate3'> $ 49.99</p>
                    <p>Seats Washing</p>
                    <p>Vacuum Cleaning</p>
                    <p>Exterior Cleaning</p>
                    <p>Interior Wet Cleaning</p>
                    <p>Window Wiping</p>
                    <button id='btn-price3'>Book Now</button>
                </div>
            </div>
            <br/>
            <br/>
            
        </div>
    );
}

export default Price;