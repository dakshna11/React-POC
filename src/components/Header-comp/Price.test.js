import React from "react";
import { shallow } from "enzyme";
import Price from "./Price";

const price = 'Test Price'
let wrapped = shallow(<Price>{price}</Price>)

describe('Price', ()=>{
    it('should render the Price Component correctly', ()=>{
        expect(wrapped).toMatchSnapshot()
    })
})