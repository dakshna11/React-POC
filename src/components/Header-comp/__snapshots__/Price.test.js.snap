// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Price should render the Price Component correctly 1`] = `
<div>
  <div
    className="about-us"
  >
    <div
      className="about-container"
    >
      <h1
        className="about-text"
      >
        Washing Plan
      </h1>
      <p>
        <Link
          className="wordsWP"
          to="/Price"
        >
          Home 
        </Link>
        /
        <Link
          className="wordsWP"
          to="/Price"
        >
           Price
        </Link>
      </p>
    </div>
  </div>
  <div>
    <p
      id="washing-heading"
    >
      Washing Plan
    </p>
    <h1
      style={
        Object {
          "color": "#202c45",
          "fontSize": "50px",
          "textAlign": "center",
        }
      }
    >
      Choose Your Plan
    </h1>
  </div>
  <div
    className="price-main"
  >
    <div
      className="price1"
    >
      <p
        id="price-heading"
      >
        BASIC CLEANING
      </p>
      <p
        id="price-rate1"
      >
         $ 25.99
      </p>
      <p>
        Seats Washing
      </p>
      <p>
        Vacuum Cleaning
      </p>
      <p>
        Exterior Cleaning
      </p>
      <p>
        Interior Wet Cleaning
      </p>
      <p>
        Window Wiping
      </p>
      <button
        id="btn-price1"
      >
        Book Now
      </button>
    </div>
    <div
      className="price2"
    >
      <p
        id="price-heading"
        style={
          Object {
            "color": "red",
          }
        }
      >
        PREMIUM CLEANING
      </p>
      <div
        id="price-rate2"
      >
        <p>
           $
        </p>
        <p>
           35
        </p>
        <p>
          .99
        </p>
      </div>
      <p>
        Seats Washing
      </p>
      <p>
        Vacuum Cleaning
      </p>
      <p>
        Exterior Cleaning
      </p>
      <p>
        Interior Wet Cleaning
      </p>
      <p>
        Window Wiping
      </p>
      <button
        id="btn-about"
      >
        Book Now
      </button>
    </div>
    <div
      className="price3"
    >
      <p
        id="price-heading"
      >
        COMPLEX CLEANING
      </p>
      <p
        id="price-rate3"
      >
         $ 49.99
      </p>
      <p>
        Seats Washing
      </p>
      <p>
        Vacuum Cleaning
      </p>
      <p>
        Exterior Cleaning
      </p>
      <p>
        Interior Wet Cleaning
      </p>
      <p>
        Window Wiping
      </p>
      <button
        id="btn-price3"
      >
        Book Now
      </button>
    </div>
  </div>
  <br />
  <br />
</div>
`;
